### DISCUSSION TITLE
Is there a different installation procedure for shared hosting (Cannot use Softoculous)?

### DISCUSSION THREAD

**Discussion by djimatr** [YetiForceCompany/YetiForceCRM#17435] [How to]
On Virtualmin with a /home/websitename/public_html configuration - can set the php versionper website utilizing php fpm. I've tried installing several versions, 6.4, 6.5 and 7 all unsuccessfull. The last version I tried was 6.4 and got to the end - gateway timeout even though I have execution time to 600 sec. Just recieve invalid session even after I clear browser. Even though the configuration states that everything is good it still fails.

**Comment by MonsterITServices** [comment]
Virtualmin will not work with yetiforce. sorry to say i spent over 50 hours playing with it trying to get to work but virtualmin will restrict it and not let you install yetiforce because of sec_mod. 
I would install on a new server that's not running virtualmin

**Comment by djimatr** [comment]
Oh I see, sec_mod isn’t in the requirement list during install – so will it not work without it? I tried php 7.4 I got as far as invalid session once() tried clearing the browser cache but still invalid session) and only on yetiforce 6.4 – it creates a lot of db tables – 710 tables I believe but can’t get much further – always get a gateway timeout during the last part of the install.

Some other things I noticed: during the install requirements it says: config/Modules  No – when I look in the config there is no Modules dir so I copy the modules directory in there manually – that can’t be right.

Strange but thanks for trying.

**Reply by MonsterITServices** [reply]
So no sec_mod, and yes create the folder manually.
Which os you using?  Go to command line and type php -v and then php -m and print output.

**Reply by djimatr** [reply]
Alma linux 9.2 
PHP 8.1.28 (cli) (built: Apr 10 2024 05:53:50) (NTS gcc x86_64)
Copyright (c) The PHP Group
Zend Engine v4.1.28, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.28, Copyright (c), by Zend Technologies
[PHP Modules]
apcu
bcmath
bz2
calendar
Core
ctype
curl
date
dom
exif
fileinfo
filter
ftp
gd
gettext
hash
iconv
imap
intl
json
ldap
libxml
mbstring
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
readline
Reflection
session
SimpleXML
soap
sockets
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter
xsl
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache

**Reply by MonsterITServices** [reply]
okay, seems okay module wise. I would check over the /etc/php/8.1/cli/php.ini file and make sure you have everything you need value wise entered correctly then try this tutorial:
(missing the mysql install step)
[https://docs.vultr.com/install-yetiforce-on-ubuntu-20-04](url)

**Reply by djimatr** [reply]
Thanks for trying but I gave up on it. The licensing is too restrictive for me anyway - I just went with vtiger v8 instead and it installed no problem.
